lib/wasix/src/os/task/process.rs

```diff
@@ -534,6 +570,147 @@ impl WasiProcess {
             thread.set_status_finished(Ok(exit_code))
         }
     }
```

Comment:
Should we have an exposed API to modify the backoff behaviour at runtime?

Reply:
I don't think we'll need it - so best we avoid the complexity and cost of doing this - its quite a bit harder to wake up threads should the backoff be changed and it would be unexpected behavior if it didn't wake them up when invoking the API